net-print/cnrdrvcups-lb/cnrdrvcups-lb-5.40-r2.ebuild

```diff
@@ -111,6 +114,10 @@ src_prepare() {
 
 	# For some reason, @AR@ is defined everywhere else correctly, but not here.
 	sed -i -e "s/AR = ar/AR = $(tc-getAR)/g" ./cnrdrvcups-common-${PV}/buftool/Makefile.in || die
+
+	pushd "${S}/cnrdrvcups-lb-${PV}" || die
+	eapply "${FILESDIR}/${PN}-fix-the-cast-error.patch"
+	popd || die
```

Comment:
You could simply adjust the patch file to apply on correct path, but this is fine too.